components/footer.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="w-full bg-roses-dark py-10">
+    <div class="container mx-auto flex flex-col gap-12">
+      <div class="flex flex-col lg:flex-row justify-between items-start">
+        <div class="order-2 lg:order-1">
+          <img
+            src="https://assets-cdn.sums.su/YU/website/img/Roses/Roses_are_white_footer.png"
+            alt=""
+          />
+        </div>
+        <div
+          class="flex mb-10 lg:mb-0 text-white text-xs lg:text-sm gap-10 order-1 lg:order-2"
+        >
+          <div class="flex flex-col gap-2 lg:gap-1">
+            <h2 class="text-roses-red mb-2">Useful Links</h2>
```

Comment:
Nothing actionable but I'm not sure if we need to duplicate the nav bar here but we can think about it